### ISSUE TITLE
There should be an acknowledgement section in the README for the old NWNX

### ISSUE THREAD

**Issue by jd28** [nwnxee/unified#582]
Clearly work here was inspired/influenced by it.  There's some "You're awesome, Can I give you money", section which I won't comment on.  So I can't see acknowledging the past would be a problem.  Some of the plugins were probably copied (from GPL code, but whatever).  Why not give some recognition to Virusman, Dumbo, etc, etc?

**Comment by mtijanic** [comment]
To be fair, the core NWNX was rewritten from scratch by @Liareth (my changes built on top of that) , and it's not up to us to acknowledge what inspired or helped her write it. I'd leave it up to her to write that section, if she wants to.

I'm totally game to add an acknowledgement section into every plugin's readme and populate where appropriate. Also, plugins can technically be GPL'd if needbe, so if any code was copied directly (I think JVM might have been), we should change that license.

**Comment by Liareth** [comment]
As Sherincall said, I wrote the core from scratch without referencing 1.69's NWNX. Most of the plugins were influenced by 1.69 insofar as they exist to solve the same problem that users have, but - at least when I was writing them - the API is usually different because I ignored the 1.69 API and implemented it the way that made the most sense to me at the time.

The decision isn't mine on whether an acknowledgement is appropriate now because I'm not actively maintaining the project. Even if I was, there's a ton of active contributors and more plugin code than core code. The decision lies with them about whether they feel the project is similar enough to 1.69 to warrant an acknowledgement.

In my view, the code I wrote was not similar enough to 1.69 to warrant an acknowledgement, which is why I didn't add one at the time. The main similarity comes from the name of the project. I empathise that the team behind 1.69 NWNX may feel usurped because of the naming similarity and would support an acknowledgement on that basis alone.

I don't see any harm in adding an acknowledgement, but I would find it challenging to write one because it would feel like I was pandering for goodwill. If I were maintaining the project, and somebody else were to write an appropriate acknowledgement, I would merge it.

The `You're awesome, Can I give you money` section is there because people kept offering to donate to me or asking me to start a Patreon, and I didn't want to accept money for the project. People have stopped asking since I added that section. If the active contributors find it distasteful, they can remove it.
